app/templates/macros/toolkit_forms.html

```diff
@@ -1,3 +1,14 @@
+{#
+The following form macros extend on top of of the macros found in the front end toolkit so we can support additional
+functionality needed for forms for applying for a framework.
+
+So we can parse question references for the supplier framework declaration forms, we redefine all our individual
+questions here with parsed error messages, hints and question.
+
+So we can format the value of a form upload to be a 'pretty' datetime string for file uploads for editing a
+service, we set a custom filter on its value.
```

Comment:
We absolutely ought to try and be consistent in how we display dates and times, so this one seems like a less good reason for us to have something custom here.

Reply:
(Oh no, wait what: it's extracting a date from a URI?? I don't even want to know. *clicks approve*)